hardware/plugins/DelayedLink.h

```diff
@@ -253,7 +253,7 @@ namespace Plugins {
 						while (!shared_lib_ && (ent = readdir(lDir)) != NULL)
 						{
 							std::string filename = ent->d_name;
-							if (dirent_is_directory(szLibrary, ent) && (filename.length() > 2))
+							if ((ent->d_type == DT_DIR) && (filename.length() > 2))
```

Comment:
Please keep dirent_is_directory, it supports symlinks also, not only directories